common/src/main/java/org/apache/hertzbeat/common/config/CommonProperties.java

```diff
@@ -227,6 +238,73 @@ public String getTemplateId() {
             return templateId;
         }
 
+        public void setTemplateId(String templateId) {
+            this.templateId = templateId;
+        }
+    }
+    public static class AliYunSmsProperties {
+
+        /**
+         * 阿里云账户访问密钥id
+         */
```

Comment:
👍 hi, please use the english describe.

Reply:
Should all the descriptions inside be written in English?